api/v1/models/base.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+""" Base
+"""
+from sqlalchemy import (
+        Column,
+        Integer,
+        ForeignKey,
+        Table
+        )
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from datetime import datetime
+
+
+Base = declarative_base()
+
+user_organisation_association = Table('user_organisation', Base.metadata,
```

Comment:
We'll stick to en-us across the boilerplates. 
So organization instead of organisation